libs/community/langchain_community/document_loaders/pebblo.py

```diff
@@ -83,6 +88,23 @@ def __init__(
         )
         self.pb_client.send_loader_discover(self.app)
 
+    @classmethod
```

Comment:
Note: This can work with multiple data sources without requiring any changes on the app side. It also supports threads. 
**App-side changes are only necessary when using multi-processesing to run loaders.**